faezeh_calculator.rb

```diff
@@ -0,0 +1,64 @@
+# When a user runs this program, the program should ask the user for the following things:
+
+# 1- a math operation
+# 2- a number for the math operation
+# 3- a second number for the math operation
+
+puts "I am glad you chose my calculator, because it's awesome!
+Which operator woudl you like to use?
+1. add(+)
+2. subtract(-)
+3. multiply(*)
+4. devide (/)
+Please choose one operator (name or symbol)"
+
+command = gets.chomp.downcase
+
+# operators = ["add", "+", "subtract", "-",
+#  "multiply", "*", "devide", "/"]
+
+# operators.each do |i|
+#   puts i + "(" #{i+1} + ")"
+# end
+
+until ["add", "+", "subtract", "-", "multiply", "*", "divide", "/", ].include?(command)
+  puts "What you entered is not an option! 
+  Please tell me to add (+), subtract (-) multiply (*) or devide (/)!"
+  command = gets.chomp.downcase
+end
+
+class Object
+  def is_number?
+    to_f.to_s == to_s || to_i.to_s == to_s
+  end
+end
```

Comment:
Clever.  Did you find this on Stack Overflow or someplace.  It's best to leave a comment with a link to where you find something like this.

Reply:
I had found it from stack overflow when I was writing code for the technical challenge. I will comment the source from now on when I find something. thank you for reminding me.